javascript-practice/src/styles/sections/add-user.css

```diff
@@ -0,0 +1,95 @@
+.popup-overlay-active {
```

Comment:
All style for popup or add-user? I don't think we can reuse this style file. Pls consider to split it